bootstrap.php

```diff
@@ -137,6 +140,33 @@ function nfd_create_event_queue_table() {
 	dbDelta( $sql );
 }
 
+/**
+ * Update affiliation code in option table
+ *
+ * @param object $container Container information
+ */
+function nfd_update_options_table( $container ) {
+	$brand_code = array(
+		'bluehost'        => '86241',
+		'hostgator'       => '57686',
+		'web'             => '86239',
+		'crazy-domains'   => '57687',
+		'hostgator-india' => '57686',
+		'bluehost-india'  => '86241',
```

Comment:
We don't have a good way to determine if a site is Bluehost India or HostGator India right now. We can keep this code, but it won't do anything.